src/Console/StatusCommand.php

```
@@ -38,9 +37,7 @@ public function handle(Core $nightwatch): int
         }
 
         try {
-            if (! $nightwatch->ingest->ping()) {
-                throw new RuntimeException('Failed to check the status of the Nightwatch agent');
-            }
+            $nightwatch->ingest->ping();
```

This throws an exception on failure instead of returning a bool.